code/development/getci.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import division
+import numpy as np
+import math
+from scipy import stats
+import nibabel as nib
+import numpy.linalg as npl
+import roi_extraction
+
+file_name_con = "sub011_task001_run001_func_data_mni.nii.gz"
+file_name_scz = "sub001_task001_run001_func_data_mni.nii.gz"
+img_con = nib.load(file_name_con)
+img_scz = nib.load(file_name_scz)
+data_con = img_con.get_data()[..., 5:]
+data_scz = img_scz.get_data()[..., 5:]
+
+def roi_cor (data, roi1,roi2):
+#input: 
+	# roi1 and roi2 are two list of tuples indicating the voxel 
+	# indexes of the ROI1 and ROI2 respectively
```

Comment:
You may use python docstring (e.g. """ comments """) instead of comments (e.g. # comments) immediately below method names. Also, the spacing is a little bit werid.